pkg/registry/daemoncontroller/rest.go

```diff
@@ -78,9 +76,10 @@ func (dcStrategy) ValidateUpdate(ctx api.Context, obj, old runtime.Object) field
 // ControllerToSelectableFields returns a label set that represents the object.
 func ControllerToSelectableFields(controller *api.DaemonController) fields.Set {
 	return fields.Set{
-		"metadata.name":               controller.Name,
-		"status.nodesRunningDaemon":   strconv.Itoa(controller.Status.NodesRunningDaemon),
-		"status.nodesShouldRunDaemon": strconv.Itoa(controller.Status.NodesShouldRunDaemon),
+		"metadata.name":                 controller.Name,
+		"status.currentNumberScheduled": strconv.Itoa(controller.Status.CurrentNumberScheduled),
+		"status.numberMisscheduled":     strconv.Itoa(controller.Status.NumberMisscheduled),
+		"status.desiredNumberScheduled": strconv.Itoa(controller.Status.DesiredNumberScheduled),
```

Comment:
(After IRL conversation-- maybe these can be used to wait for it to be safe to delete? I guess leave them.)